### PR TITLE
Minor bugs fixed

### PR DESCRIPTION
**Pull request by rohitkori**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -153,13 +153,13 @@ function BMCLinkScheduler() {
...
-    rule.hour = rule.hour - 5;
-    rule.minute = parseInt(rule.minute) - 30;
-  }
+  // if (rule.minute < 30) {
+  //   rule.hour = rule.hour - 1;
+  //   rule.minute = parseInt(rule.minute) + 30;
+  // } else {
+  //   rule.hour = rule.hour;
+  //   rule.minute = parseInt(rule.minute) - 30;
+  // }
```
Comment (RahulGopathi) [inline app.js]: These aren't needed anymore.remove them

File: app.js [hunk 2]
```diff
@@ -183,6 +183,33 @@ function resetScheduler() {
...
+//     } else {
+//       console.log(data.result.reason)
+//     }
+//   } else {
+//     console.log(error, body)
+//   }
+// }
+
+// request(options, callback)
+
```
Comment (RahulGopathi) [inline app.js:179]: The Metadata API is taking more time right. There is no need for this anymore I feel

File: package.json [hunk 3]
```diff
@@ -6,7 +6,10 @@
     "fs": "^0.0.1-security",
     "lodash": "^4.17.21",
     "node-cron": "^3.0.0",
-    "node-schedule": "^2.0.0"
+    "node-schedule": "^2.0.0",
+    "nodemon": "^2.0.16",
+    "request": "^2.88.2",
+    "url-metadata": "^2.5.0"
```
Comment (RahulGopathi) [inline package.json]: you can remove these packages as they aren't required anymore

File: app.js [hunk 4]
```diff
@@ -260,18 +287,27 @@ client.on("message", (msg) => {
     if (setTimeCommand[2] == "days") {
       try {
         if (
-          setTimeCommand.lenght !== 5 ||
-          setTimeCommand[3] == "" ||
-          setTimeCommand[4] == ""
+          // setTimeCommand.lenght != 5
+          // setTimeCommand[3] == "" ||
+          // setTimeCommand[4] == ""
```
Comment (RahulGopathi) [inline app.js]: remove these comments

File: app.js [hunk 5]
```diff
@@ -286,31 +322,37 @@ client.on("message", (msg) => {
           msg.channel.send(
             "```Please specify time in [hours]:[minutes] format ```"
```
Comment (RahulGopathi) [inline app.js]: we can tell users to write it in 24hr format

**Review by RahulGopathi — CHANGES_REQUESTED** [review]

File: app.js [hunk 6]
```diff
@@ -253,25 +247,37 @@ client.on("message", (msg) => {
       }
     }
   }
+  if (msg.content === "get help" || msg.content === "get article") {
+    msg.reply("```Are you trying to call me?```")
+  }
 
   if (msg.content.startsWith(prefix + "set article ")) {
     correctTimeProvided = false;
+    displayTime = true;
```
Comment (RahulGopathi) [inline app.js]: What is this `displayTime`?

Comment (rohitkori) [reply app.js]: It's a boolean variable which will not display time in case of any error  in set time and get time commands

Comment (RahulGopathi) [reply app.js]: We aren't displaying the time anywhere right, better change the variable name or write a comment there

Comment (rohitkori) [reply app.js]: okk i will do it

**Review by RahulGopathi — APPROVED** [review]

**Pull request merged**